content/pages/burials-and-memorials/eligibility.md

```diff
@@ -55,3 +60,13 @@ We give the surviving spouse, children, and parents of deceased Servicemembers a
 Yes. You can plan ahead to make the process of applying for a burial in a VA national cemetery easier for your family members in the event of your death. To do this, you’ll need to apply for a pre-need eligibility determination. [Find out how to apply](/burials-and-memorials/pre-need/).
 
 **Please note:** The pre-need eligibility determination is only for national cemeteries. If you want to be buried in a state or tribal Veterans cemetery, check the site you want ahead of time, since some state Veterans cemeteries require that you live in that state or have other rules around eligibility. [Find a state Veterans cemetery](http://www.cem.va.gov/cem/cems/listcem.asp).
+
+### Ready to apply?
+
```

Comment:
This entire section has been added, was it not here for a specific reason before? Should it not be added now?